src/main/java/com/robothand/highqualitybot/Config.java

```diff
@@ -11,30 +14,27 @@
  * Reads in properties from a specified configuration file, and provides access to the values.
  */
 public class Config {
-    private final File configFile;
-
-    public String TOKEN;
+    String TOKEN;
     public String PREFIX;
+    public String OWNERID;
+    public ArrayList<Command> ALLOWED;
+    public ArrayList<Command> DISALLOWED;
+    public boolean ALLOWEDHASPRECEDENCE;
+    public ArrayList<String> PERMGROUPS;
 
 
     public Config(String path) throws FileNotFoundException {
```

Comment:
I'd suggest making this more like a utility class with static member variables and something like a static `getConfig` method. Then rather than having the `Bot` class hold a static reference to one instance of this class, we could use `Config.PROPERTY_WE_NEED` in place of all our `Bot.config.PROPERTY` calls. Let me know what your thoughts are.